dashboards-observability/public/components/custom_panels/custom_panel_view.tsx

```diff
@@ -65,7 +65,7 @@ import { PPLReferenceFlyout } from '../common/helpers';
 
 interface Props {
   panelId: string;
-  page?: string;
+  page: 'app' | 'operationalPanels';
```

Comment:
should this be `page: string`. Does interface take in values?

Reply:
any reason why was this changed from being optional to necessary?

Reply:
I think it's safer to make sure that the page is either one of those, or you can add a new one there so the string isn't misspelled or something. I changed it from being optional to necessary because leaving it optional meant it was forgotten to be included as a prop in some places and that lead to incorrect UI.